### DISCUSSION TITLE
How to utilize 'right side panel' as shown in https://keystonejs.com/why-keystone

### DISCUSSION THREAD

**Discussion by jerrami** [keystonejs/keystone#7447] [Questions] [answered]
My first post here, so please go easy on me :)

I've just recently added fields to most of my content types such as creator, created time, etc.  I absolutely want to do something similar to the screenshot shown in the [Why Keystone?](https://keystonejs.com/why-keystone) article.  Specifically how the `Post details` section has creator, etc. listed under it.  Looking through the documentation, I haven't yet found the secret sauce to unlock adding fields to that panel.  Can someone please point me in the right direction and if there's any sample code, that'd be even better!  Keystone 6 btw, if that's part of why I haven't found it yet.

![image](https://user-images.githubusercontent.com/21203507/162260119-f2472d85-e3a8-4417-b32c-6e8faaccfa4e.png)

Thanks in advance!

**Comment by raveling** [comment, marked as answer]
@jerrami at this point in time we don't have an easy way to populate the item view sidebar with that metadata. We've updated the image on the site to reflect the current Admin UI design. Sorry for the confusion!

**Reply by jerrami** [reply]
Thanks for following up and letting me.
